DVCodingStyle.md

```diff
@@ -626,14 +718,14 @@ cause race conditions if there are any parallel threads.
     phase.drop_objection(this, $sformatf("%s objection dropped", `gfn));
     ```
 
-8.  It is recommended, but not necessary, to use the built-in UVM timeout
-    mechanism, `uvm_root::set_timeout(<time_limit>, 1)`.
+2.  It is recommended to use the built-in UVM timeout mechanism,
+    `uvm_root::set_timeout(<time_limit>, 1)`.
     The base test in every testbench should specify a default timeout limit,
     which can be overridden by every derivative test.
     A standard plusarg `+UVM_TIMEOUT=...` is recommended to be used for this
     override mechanism.
-    Alternatively, some separate timeout mechanism can be created and utilized
-    within the testbench.
+    Alternatively, some separate global timeout mechanism can be created and
+    utilized within the testbench.
```

Comment:
Do we have examples of tests that use a different timeout mechanism? If not, maybe we should drop this exception - we already mandate that UVM tests derive from the `dv_base_...` infrastructure that provides the suggested version, no?

Reply:
I agree with @rswarbrick unless we already have code in place that deviates from this suggestion we should enforce the use of the build in timeout

Reply:
That's true, I believe the existing approach is done using `::set_timeout(...)`, I'll delete the second part.